content/cpp/concepts/math-functions/terms/round/round.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'round()'
+Description: 'Returns the integer that is nearest to the argument, with halfway cases rounded away from zero.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `round()` function returns the integer that is nearest to the argument, with halfway cases rounded away from zero.
+
+## Syntax
+
+```pseudo
+round(num);
+```
+
+The `num` parameter must be a `double`,`float`, or `long double`. The return value will be an integer.
```

Comment:
I would maybe include one more sentence here that explains what a halfway case is and how they are used in the `round()` function.